styles.css

```diff
@@ -347,11 +352,176 @@ hr {
 }
 
 @media only screen and (min-width: 768px) {
-  .menu-container,
-  .main,
-  .grid-container,
-  .about-section,
+  .menu-container {
+    padding: 10px 80px;
+    position: fixed;
+    width: 100%;
+  }
+
+  .navbar img {
+    display: none;
+    flex: 1 1 200px;
+  }
+
+  .nav-list {
+    margin-left: 50rem;
+    float: right;
+    list-style: none;
+    display: flex;
+    gap: 10px;
+  }
+
+  .nav-list a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .main {
+    background-image: url('./images/header-shapes\ mobile@2x.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    padding: 10rem 10rem;
+    height: 90vh;
+  }
```

Comment:
Going with the figma design, the background of the hero section is supposed to span up to 916px on a screen width of 1440px. Kindly adjust your design to match.

Your design:
![image](https://user-images.githubusercontent.com/20554583/236247650-08f9ef28-8a9c-4cc2-9b80-750cb351e098.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/236247812-b7248162-cc8c-4317-a0f5-62a56425bd21.png)